DataFormats/Detectors/TPC/src/DataFormatsTPCLinkDef.h

```diff
@@ -26,5 +26,6 @@
 #pragma link C++ class o2::TPC::TPCClusterFormatHelper + ;
 #pragma link C++ class o2::TPC::TrackTPC + ;
 #pragma link C++ class o2::TPC::Cluster + ;
+#pragma link C++ class std::vector < o2::TPC::Cluster > +;
```

Comment:
probably makes sense ... but could be extended to other classes like TPC::TrackTPC etc. to be consistent;

Reply:
I had the same idea right after creating the PR, update is pushed